src/constants.zig

```diff
@@ -298,7 +298,7 @@ pub const connection_delay_min_ms = config.process.connection_delay_min_ms;
 pub const connection_delay_max_ms = config.process.connection_delay_max_ms;
 
 /// The maximum number of outgoing messages that may be queued on a replica connection.
-pub const connection_send_queue_max_replica = std.math.max(std.math.min(clients_max, 4), 2);
+pub const connection_send_queue_max_replica = @max(@min(clients_max, 4), 2);
```

Comment:
After this PR we should add `stdx.clamp`, the minmax combo is too easy to mess up,